### PR TITLE
reversed option for close all button

### PR DESCRIPTION
**Pull request by ljmerza**
created a closerReversed option to prepend the close all button to the top of a jgrowl messages

### CODE REVIEW

**Comment by stanlemon** [comment]
Can you tell me a bit more about your use case?

**Comment by ljmerza** [comment]
sometimes we have many jgrowls appearing causing the close all button to be pushed out of the document view. To ensure the close all button is always view-able, we keep it at the top.

File: jquery.jgrowl.js [hunk 1]
```diff
@@ -214,6 +214,7 @@
 			openDuration:		'normal',
 			easing:				'swing',
 			closer:				true,
+			closerReversed:		0,
```
Comment (stanlemon) [inline jquery.jgrowl.js]: Would you be willing to rename this to closerGlue and match the after/before semantics of the glue property?  They seem closely related to me.

**Comment by ljmerza** [comment]
renamed to closerGlue

**Comment by stanlemon** [comment]
Thanks @ljmerza can you switch the value to be 'before' and 'after' so it aligns with the other glue option?

Also, if you can just go ahead and backout the source map and min versions, I'll run those with package config updates and other changes in master.

**Comment by stanlemon** [comment]
Closing due to inactivity.

**Pull request closed without merging**